src/typing.py

```diff
@@ -1456,7 +1460,9 @@ def get_type_hints(obj, globalns=None, localns=None):
             isinstance(obj, types.FunctionType) or
             isinstance(obj, types.BuiltinFunctionType) or
             isinstance(obj, types.MethodType) or
-            isinstance(obj, types.ModuleType)
+            isinstance(obj, types.ModuleType) or
+            isinstance(obj, _wrapper_descriptor_type) or
+            isinstance(obj, _method_wrapper_type)
```

Comment:
Thank, you! I have already noticed this some time ago, I would prefer to fix this separately after this PR is merged.